### PR TITLE
ServiceNow - Added the support of custom fields to create and update commands

### PR DESCRIPTION
**Pull request by dem-evgeny**
Added the support of custom fields to create and update commands

### CODE REVIEW

File: Integrations/integration-ServiceNow.yml [hunk 1]
```diff
@@ -292,18 +298,46 @@ script:
...
         return ( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': res.obj.result, "HumanReadable": md} );
     }
 
+
+    var splitCusFilds = function(cusFields){
+        var dicFields = {};
+        var arrFields = cusFields.split(';');
+
+        for(var i=0; i<arrFields.length; i++){
+            if (arrFields[i].split('=').length > 1){
```
Comment (meirwah) [inline Integrations/integration-ServiceNow.yml]: res = arrFields[i].split('=')

File: Integrations/integration-ServiceNow.yml [hunk 2]
```diff
@@ -96,11 +95,11 @@ script:
         }
     };
 
-    var getBody = function(templateName) {
+    var getBody = function(templateName, cusFields) {
         var body = {};
         var tmpRes;
 
-        if (templateName){
+        if (templateName !== undefined){
```
Comment (meirwah) [inline Integrations/integration-ServiceNow.yml]: ?

File: Integrations/integration-ServiceNow.yml [hunk 3]
```diff
@@ -1360,4 +1398,4 @@ script:
     description: Upload new file
   isfetch: true
 hidden: false
-releaseNotes: "Support templates on ticket/incident creation"
\ No newline at end of file
+releaseNotes: Added support for custom fields in commands: create and update.
```
Comment (meirwah) [inline Integrations/integration-ServiceNow.yml]: fails on ymal validation

File: Integrations/integration-ServiceNow.yml [hunk 4]
```diff
@@ -3,7 +3,6 @@ commonfields:
   version: -1
 name: ServiceNow
 display: ServiceNow
--fromversion: 2.5.0
```
Comment (dorsha) [inline Integrations/integration-ServiceNow.yml]: why did you remove this fromversion?

Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: Didn't removed it.

Comment (dorsha) [inline Integrations/integration-ServiceNow.yml]: ok, so bring it back

File: Integrations/integration-ServiceNow.yml (hunk 3 above)
Comment (meirwah) [inline Integrations/integration-ServiceNow.yml]: maybe write here full commands name:  servicenow-update, servicenow-create

Comment (meirwah) [inline Integrations/integration-ServiceNow.yml]: by the way those names, are not accurate.. better is to say:
```servicenow-create-ticket``` i think...

Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: I am not sure that is is good to change the names now because playbooks already use this names.

File: Integrations/integration-ServiceNow.yml (hunk 2 above)
Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: Changed back

File: Integrations/integration-ServiceNow.yml (hunk 1 above)
Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: Changed

File: Integrations/integration-ServiceNow.yml (hunk 3 above)
Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: Changed

File: Integrations/integration-ServiceNow.yml [hunk 5]
```diff
@@ -1398,4 +1400,4 @@ script:
     description: Upload new file
   isfetch: true
 hidden: false
-releaseNotes: Added support for custom fields in commands: create and update.
+releaseNotes: "Added support for custom fields in commands(servicenow-update and servicenow-create)."
```
Comment (dorsha) [inline Integrations/integration-ServiceNow.yml]: You are missing a space between the commands and the (

Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: Fixed

File: Integrations/integration-ServiceNow.yml [hunk 6]
```diff
@@ -1027,6 +1065,8 @@ script:
     - name: id
       required: true
       description: Id(UID) of the ticket to update
+    - name: custom_fields
+      description: 'Custom fields in this format: fieldname=value;filedname=value...'
```
Comment (gshriki) [inline Integrations/integration-ServiceNow.yml]: fix typo: filedname -> fieldname

File: Integrations/integration-ServiceNow.yml [hunk 7]
```diff
@@ -724,6 +760,8 @@ script:
     - name: priority
     - name: template
       description: Template name to use as a base to create new incidents.
+    - name: custom_fields
+      description: 'Custom fields in this format: fieldname=value;filedname=value...'
```
Comment (gshriki) [inline Integrations/integration-ServiceNow.yml]: filedname -> fieldname

Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: fixed

File: Integrations/integration-ServiceNow.yml (hunk 6 above)
Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: fixed
